misc/README.md

```diff
@@ -7,6 +7,9 @@ The process for updating the `make e2e`'s input is manual.  Each time a go-algor
 | Upload test run artifacts to S3.<br/>* Publishing to S3 streamlines running the tests for other developers and CI.<br />* If you don't have an AWS account, request artifact upload by pinging Will, Michael, or DevOps.                                                       | Run the s3 copy command provided by `misc/buildtestdata.sh` output.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
-| `make e2e` runs on every PR. Validate results on the next CI run.                                                                                                                                                                                                              | * Find the most recent workflow starting _after_ updating the S3 bucket:  [https://app.circleci.com/pipelines/github/algorand/indexer?branch=develop&filter=all](https://app.circleci.com/pipelines/github/algorand/indexer?branch=develop&filter=all).<br/>* If successful, confirm the test used the artifact you uploaded.  Click a job link, expand _make e2e_ and search for `INFO:util:s3://`.<br />* If unsuccessful, investigate source of error.  If additional help is needed, raise in #indexer.  It is _your_ responsibility to ensure the build is fixed. |
\ No newline at end of file
+| `make e2e` runs on every PR. Validate results on the next CI run.                                                                                                                                                                                                              | * Find the most recent workflow starting _after_ updating the S3 bucket:  [https://app.circleci.com/pipelines/github/algorand/indexer?branch=develop&filter=all](https://app.circleci.com/pipelines/github/algorand/indexer?branch=develop&filter=all).<br/>* If successful, confirm the test used the artifact you uploaded.  Click a job link, expand _make e2e_ and search for `INFO:util:s3://`.<br />* If unsuccessful, investigate source of error.  If additional help is needed, raise in #indexer.  It is _your_ responsibility to ensure the build is fixed. |
+
+# Help
```

Comment:
@algoidurovic Optionally, I suggest a more specific header name.  If you prefer to keep as is, you're welcomed to resolve the thread.

If you accept the change, then I'd also change the table reference to explicitly refer to the section.  I suggest the explicit reference because it's possible to imagine other sections unrelated to `e2elive.py` will be added over time.  


```suggestion
## e2elive.py prerequisites
```